.ci/travis/linux/docker-build-test.sh

```diff
@@ -145,9 +145,9 @@ EOT
 # Run tests
 ###########
 CURRENT_TIME=$(date +%s)
-TIMEOUT=$((( TRAVIS_TIME - UPLOAD_TIME) * 60 - CURRENT_TIME + TRAVIS_TIMESTAMP))
+TIMEOUT=500 #$((( TRAVIS_TIME - UPLOAD_TIME) * 60 - CURRENT_TIME + TRAVIS_TIMESTAMP))
```

Comment:
the rationale about this change?

Reply:
@luipir , will be reverted asap. It is to make sure my test time is issue. right now travis compute time to 60s for running 3 otb algs while TIMEOUT is set to 55s

Reply:
@luipir , do you know if I can rebuild qgis docker image to add otb package?